src/panels/calendar/dialog-calendar-event-detail.ts

```diff
@@ -0,0 +1,457 @@
+                <div id="date-range-details-content">
+                  <ha-formfield
+                    .label=${this.hass.localize(
+                      "ui.components.calendar.event.all_day"
+                    )}
+                  >
+                    <ha-switch
+                      id="all_day"
+                      .checked=${this._allDay}
+                      @change=${this._allDayToggleChanged}
+                    ></ha-switch>
+                  </ha-formfield>
+
+                  <ha-rrule
+                    .hass=${this.hass}
+                    .value=${this._data!.rrule}
+                    .mutable=${true}
+                    @value-changed=${this._handleRRuleChanged}
+                  >
+                  </ha-rrule>
+
+                  <ha-combo-box
+                    name="calendar"
+                    .hass=${this.hass}
+                    .disabled=${!this._params.calendarId}
+                    .label=${this.hass.localize("ui.components.calendar.label")}
+                    .value=${this._calendarId!}
+                    .renderer=${rowRenderer}
+                    .items=${this._calendars}
+                    item-id-path="entity_id"
+                    item-value-path="entity_id"
+                    item-label-path="name"
+                    required
+                    @value-changed=${this._handleCalendarPicked}
+                  ></ha-combo-box>
+                </div>
+
+                <mwc-button
+                  slot="primaryAction"
+                  @click=${this._createEvent}
+                  .disabled=${this._submitting}
+                >
+                  ${this.hass.localize("ui.components.calendar.event.add")}
+                </mwc-button>
+              `
+            : html`
+                <div class="field">
+                  <ha-svg-icon .path=${mdiCalendarClock}></ha-svg-icon>
+                  <div class="value">
+                    ${this._formatDateRange()}<br />
+                    ${this._data!.rrule !== undefined
+                      ? html` <ha-rrule
+                          .hass=${this.hass}
+                          .value=${this._data!.rrule}
+                          .mutable=${false}
+                        >
+                        </ha-rrule>`
+                      : html``}
+                  </div>
+                </div>
+
+                <div class="attribute">
+                  <state-info
+                    .hass=${this.hass}
+                    .stateObj=${stateObj}
+                    .inDialog="true"
+                  ></state-info>
+                </div>
+
+                <mwc-button
+                  slot="secondaryAction"
+                  class="warning"
+                  @click=${this._deleteEvent}
+                  .disabled=${this._submitting}
+                >
+                  ${this.hass.localize("ui.components.calendar.event.delete")}
+                </mwc-button>
+              `}
+        </div>
```

Comment:
The buttons should not be inside the div

Reply:
Moved.